client/src/screens/Client/Setup/ReportFinance/ReportFinance.js

```diff
@@ -160,4 +160,4 @@ export default function ReportFinance(props) {
       )}
     </AuthConsumer>
   );
-}
+}
```

Comment:
its just code formatting, no change from that file

Reply:
do not change it